cmake/DyninstCapArchDef.cmake

```diff
@@ -8,8 +8,8 @@ include_guard(GLOBAL)
 
 set(CAP_DEFINES -Dcap_dynamic_heap -Dcap_liveness -Dcap_threads)
 
-if(DYNINST_ARCH_i386)
-  set(ARCH_DEFINES -Darch_x86)
+if(DYNINST_HOST_ARCH_i386)
+  set(ARCH_DEFINES -DDYNINST_host_arch_x86)
```

Comment:
These macros don't need to be namespaced, and I prefer having macros in all uppercase. Not a requirement, but I would suggest names like `HOST_ARCH_X86`. @kupsch Thoughts?

Reply:
Yes, since we're making the change, we might as well follow normal conventions and make the names all uppercase. I would have slight preference for using the `DYNINST_` prefix since macros are global in scope to avoid the small possibility of conflict with the name in some 3rd party header file.

Reply:
@hainest @kupsch 
Won't this cause the defines in CMakefiles to have the same name as the ones in the source code?
Or is it not an issue?

Reply:
That does not matter.  `DYNINST_HOST_ARCH_X86_64` used as a cmake variable is only used in cmake, adding `-DDYNINST_HOST_ARCH_X86_64` as a compilation flag defines a C/C++ preprocessor variable.  The only time it would matter would be if the the preprocessor variable name is used in the source outside use in a string literal.